service/controller/clusterapi/v28/key/machine_deployment.go

```diff
@@ -1,26 +1,72 @@
 package key
 
 import (
+	"math/bits"
+	"net"
 	"strconv"
 
 	g8sv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
+	"github.com/giantswarm/ipam"
 	"github.com/giantswarm/microerror"
 	"sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
 
 	"github.com/giantswarm/aws-operator/pkg/label"
 )
 
+func WorkerSubnet(cr v1alpha1.MachineDeployment) string {
+	return cr.Labels[label.MachineDeploymentSubnet]
```

Comment:
Label or annotation? I would then like to have an `annotation` package and not mix these things.

Reply:
I made it an annotation.